src/Highlighter.ts

```diff
@@ -15,16 +15,20 @@ interface IOptions {
 }
 
 export default class Highlighter {
-  private container: HTMLElement;
+  private _container: HTMLElement;
```

Comment:
Does this field ever change? If not, then marking it as `readonly` should be sufficient (rather than creating a `container()`), no?

I guess I'm not sure how much value there is in enforcing read-onlyness in JS rather than just marking it in TS